src/arguments.js

```diff
@@ -67,7 +67,10 @@ async function getCommandArguments() {
             .env(ENV_VAR.EMAIL_NOTE))
         .addOption(new Option('--selfsignedcerts <flag>', 'enable or disable self-signed certicates for smtp transport')
             .default(DEFAULT_SELF_SIGNED_CERTIFICATES)
-            .choices(['true', 'false']));
+            .choices(['true', 'false']))
+        .addOption(new Option('--timeout <timeout>', 'timeout for generating report')
```

Comment:
it might be better to include the unit in the option name (`--timeout-ms`) or in the description